cmd/satellite/entrypoint

```diff
@@ -1,12 +1,32 @@
 #!/bin/sh
 set -euo pipefail
 
+SETUP_PARAMS=""
+
 if [[ -n "${API_KEY}" ]]; then
 	export STORJ_POINTER_DB_AUTH_API_KEY="${API_KEY}"
+  export STORJ_AUDIT_API_KEY="${API_KEY}"
+  export STORJ_POINTER_DB_AUTH_API_KEY="${API_KEY}"
+  export STORJ_REPAIRER_API_KEY="${API_KEY}"
+fi
+
+if [[ -n "${OVERLAY_URL}" ]]; then
```

Comment:
Don't all of the others need `:-}` as well?

https://travis-ci.org/storj/storj/jobs/478372277#L1789

Reply:
Intention was to only do those for the ones that had sane defaults. I went ahead and added to the rest of the conditionals and we can let the binary handle failure or warning.